config/global-constants/src/lib.rs

```diff
@@ -12,7 +12,8 @@
 pub const CONSENSUS_KEY: &str = "consensus";
 pub const EXECUTION_KEY: &str = "execution";
 pub const FULLNODE_NETWORK_KEY: &str = "fullnode_network";
-pub const LIBRA_ROOT_KEY: &str = "root";
+pub const LIBRA_ROOT_KEY: &str = "libra_root";
+pub const TREASURY_ROOT_KEY: &str = "treasury_root";
```

Comment:
Can we call this TREASURY_**COMPLIANCE**_KEY and similar name elsewhere?

This account is referred to as the TreasuryCompliance account everywhere else : eg: https://github.com/libra/libra/blob/master/language/stdlib/modules/Roles.move